tests/page_tests.py

```diff
@@ -5,9 +5,13 @@
 import requests
 import concurrent.futures
 
+from spellchecker import SpellChecker as PySpellChecker
 from enchant.checker import SpellChecker
```

Comment:
Should remove references to enchant and uninstall it from python if we're no longer using it.